supplier_app/tests/test_forms.py

```diff
@@ -95,8 +98,14 @@ def tearDown(self):
             rmtree('file')
 
     def _get_example_forms(self):
+        eb_entity_1 = EBEntityFactory()
+        eb_entity_2 = EBEntityFactory()
```

Comment:
mm it's necessary to add it 2 times?
Should we check that `eb_entity_1` and `eb_entity_2` could be equal? Are we checking entities?

Reply:
Change that